docs/sql-ref-syntax-qry-select-case.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: global
+title: CASE Clause
+displayTitle: CASE Clause
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+`CASE` clause uses a rule to return specific result based on the specified condition, similar to if/else statements in other programming languages.
+
+### Syntax
+
+```sql
+CASE [ expression ] { WHEN boolean_expression THEN then_expression } [ ... ]
+    [ ELSE else_expression ]
+END
+```
+
+### Parameters
+    
+* **WHEN**
+
+    Specific a boolean condition ,under which to return the `THEN` result, `WHEN` must exist in `CASE` clause.
+    
+* **THEN**
+
+    Specific a result base the `WHEN` condition, `THEN` must exist in `CASE` clause.
+    
+* **ELSE**
+
+    Specific a default result for the `CASE` rules, it is optional, if user don't use else then the `CASE` will not have default result.
+    
+* **END**
```

Comment:
We need to describe `WEHN`, `THEN`, `ELSE`, and `END`? Looks verbose to me. `boolean_expression`, `then_expression`, and `else_expression` are enough for this syntax?

Reply:
+1 for me.